### PR TITLE
It can happen that config is none. This fixes it.

### PR DESCRIPTION
**Pull request by mfalesni**
Originally part of os infra PR.

### CODE REVIEW

File: fixtures/pytest_store.py [hunk 1]
```diff
@@ -74,6 +74,10 @@ def __init__(self):
         self._user = None
 
     @property
+    def has_config(self):
+        return self.config is not None
```
Comment (seandst) [inline fixtures/pytest_store.py]: If store.config is None, then store doesn't have a config. We don't need a method for this.

**Comment by dajoRH** [comment]
Lint report for commit eaa4dded1837e40efd27438ce672653a5a424224:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: fixtures/pytest_store.py (hunk 1 above)
Comment (psav) [inline fixtures/pytest_store.py]: Agreed

File: cfme/fixtures/pytest_selenium.py [hunk 2]
```diff
@@ -325,7 +325,7 @@ def _nothing_in_flight():
         silent_failure=True)
 
     # If we are not supposed to take page screenshots...well...then...dont.
-    if not store.config.getvalue('page_screenshots'):
+    if store.has_config and not store.config.getvalue('page_screenshots'):
```
Comment (seandst) [inline cfme/fixtures/pytest_selenium.py]: So, based on the previous comment:
`if store.config and not store.config.getvalue('page_screenshots'):`

**Comment by dajoRH** [comment]
Lint report for commit ffa614397d9dfdee98788f7338b2f15a82757453:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**